server/index.js

```diff
@@ -87,4 +91,166 @@ app.post("/uploads", upload.single("myImage"), (req, res) => {
   res.json({ file: req.file });
 });
 
+//Default path
+app.get("/test", (req, res) => {
+  res.send("Test");
+});
+
+
+//-----------------------------------------------------------------------
+//Schemas
+const Schema = mongoose.Schema;
+
+//Blog post schema
+const BlogPostSchema = new Schema({
+  title: String,
+  body: String,
+  date: {
+    type: String,
+    default: Date.now()
+  }
+});
+
+//User upload schema
+// const uploadContentSchema = new Schema ({
+//   username: String,
+// });
+
+// const tempImageStorage = multer.diskStorage({
+//   destination: function(req, res, cb) {
+//     cb(null, 'uploads/')
+//   }
+// });
+
+// const multer = require('multer');
+// const upload = multer({storage: tempImageStorage});
+// router.route('img_data').post(upload.single('file'), function(req, res))
```

Comment:
For large commented sections you can switch to the /* */ commenting format to make it easier to un-comment afterwards

Reply:
Sounds good. However to uncomment these lines, you can select all lines and press CTRL + /. I will consider this for future cases though

Reply:
I've made the changes.